app/page.tsx

```diff
@@ -2,100 +2,56 @@ import Image from "next/image";
 
 export default function Home() {
   return (
-    <div className="grid grid-rows-[20px_1fr_20px] items-center justify-items-center min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]">
-      <main className="flex flex-col gap-8 row-start-2 items-center sm:items-start">
-        <Image
-          className="dark:invert"
-          src="/next.svg"
-          alt="Next.js logo"
-          width={180}
-          height={38}
-          priority
-        />
-        <ol className="list-inside list-decimal text-sm text-center sm:text-left font-[family-name:var(--font-geist-mono)]">
-          <li className="mb-2">
-            Get started by editing{" "}
-            <code className="bg-black/[.05] dark:bg-white/[.06] px-1 py-0.5 rounded font-semibold">
-              app/page.tsx
-            </code>
-            .
-          </li>
-          <li>Save and see your changes instantly.</li>
-        </ol>
+    <div className="min-h-screen p-8 pb-20 gap-16 sm:p-20 font-[family-name:var(--font-geist-sans)]">
+      <main className="max-w-4xl mx-auto">
+        <div className="text-center space-y-6 mb-12">
+          <h1 className="text-4xl sm:text-6xl font-bold tracking-tight">
+            Your Dog's Personal Chauffeur
+          </h1>
+          <p className="text-xl text-muted-foreground">
+            Safe, reliable transportation for your four-legged family members.
+            From vet visits to grooming appointments, we've got your pup
+            covered.
+          </p>
+        </div>
```

Comment:
This h1 tag is screaming out loud bro 👏🔊! Love the bold approach to ensure the page title grabs the attention 💯. However, consider making it accessible by adding an id and associating it with the page's main landmark using aria-labelledby 👀.